config/settings/test.yml

```diff
@@ -20,3 +20,7 @@ email:
   sender: "hitobito <noreply@localhost>"
   mass_recipient: "hitobito <noreply@localhost>"
   list_domain: localhost
+
+groups:
+  nextcloud:
+    enabled: true
```

Comment:
Remove? Or do tests not inherit the default settings?

Reply:
The value here is different from the default settings. 

The default is to have it **disabled**, but for tests I want it **enabled**.